Formula/neovim.rb

```diff
@@ -61,15 +61,16 @@ def install
 
     cd "deps-build" do
       ohai "Building third-party dependencies."
-      system "cmake", "../third-party", *std_cmake_args, "-DUSE_BUNDLED_BUSTED=OFF"
+      system "cmake", "../third-party", "-DUSE_BUNDLED_BUSTED=OFF", *std_cmake_args
       system "make", "VERBOSE=1"
     end
 
     mkdir "build" do
       ohai "Building Neovim."
-      system "cmake", "..", *std_cmake_args,
+      system "cmake", "..",
              "-DDEPS_PREFIX=../deps-build/usr",
-             "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+             "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+             *std_cmake_args
```

Comment:
Reordering this will compile Neovim with `Release` instead of `RelWithDebInfo`, as `CMAKE_BUILD_TYPE=Release` is part of `std_cmake_args`.